x-pack/test/functional/apps/ml/data_frame_analytics/regression_creation.ts

```diff
@@ -207,6 +207,12 @@ export default function ({ getService }: FtrProviderContext) {
           await ml.dataFrameAnalyticsResults.assertResultsTableExists();
           await ml.dataFrameAnalyticsResults.assertResultsTableTrainingFiltersExist();
           await ml.dataFrameAnalyticsResults.assertResultsTableNotEmpty();
+
+          await ml.testExecution.logTestStep('displays the map view for created job');
```

Comment:
This is currently part of the `edits the analytics job [...]` test, but I think it deserves a separate `it` block.